backend/src/controllers/master.controller.ts

```diff
@@ -58,21 +58,25 @@ export default class MasterController {
 
         if (error) return res.status(400).json(error);
 
-        const {cityId, date, time, watchSize} = req.query;
+        const { cityId, date, time, watchSize } = req.query;
         try {
-            const orders = await Order.findAll({
+            let orders = await Order.findAll({
                 where: {
-                    cityId: Number(cityId),
                     date: String(date),
-                    time: {[Op.between]: [Number(time) - 3 + 1, Number(time) + Number(watchSize) - 1]}
+                    time: { 
+                        [Op.and]:
+                            [{ [Op.gte]: +time - 3 + 1 },
+                            { [Op.lte]: +time + +watchSize - 1 }]
+                    }
                 }
             });
+            orders = orders.filter(o => o.getDataValue('time') >= +time - o.getDataValue('watchSize') + 1);
```

Comment:
This is not a good idea. It seems like a bad solution. 

Try to use sequelize.literal() in where clause.